app/build.gradle.kts

```diff
@@ -33,12 +35,10 @@ android {
             signingConfig = signingConfigs.getByName("debug")
         }
     }
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = "1.8"
+    kotlin {
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_1_8)
```

Comment:
are we using jvm 1.8 for any reason?

Reply:
No specific reason. I think it is best trying to us the minimum possible.